test/Microsoft.AspNetCore.Server.KestrelTests/FrameTests.cs

```diff
@@ -29,6 +29,7 @@ public void CanReadHeaderValueWithoutLeadingWhitespace()
             {
                 var connectionContext = new ConnectionContext()
                 {
+                    ConnectionControl = new Mock<IConnectionControl>().Object,
```

Comment:
Cool, thanks. I've actually just learned about `Mock.Of<T>` while reading @halter73's PR (#1105) 😄